driver/ppm_fillers.c

```diff
@@ -1545,6 +1545,17 @@ static int f_proc_startupdate(struct event_filler_arguments *args)
 		res = val_to_ring(args, tty_nr, 0, false, 0);
 		if (unlikely(res != PPM_SUCCESS))
 			return res;
+
+		/*
+		 * pgrp
+		 */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 24)
+		res = val_to_ring(args, (int64_t)task_pgrp_nr(current), 0, false, 0);
```

Comment:
It looks this function it's obsolete: https://github.com/torvalds/linux/blob/master/include/linux/sched.h#L1241

To me looks better to use the same functions used by `/proc/<pid>/stat`: https://github.com/torvalds/linux/blob/master/fs/proc/array.c#L476

Reply:
I didn't notice this comment, I will fix that